src/model/model.ts

```diff
@@ -73,10 +95,24 @@ export interface SerializedField<T> {
 }
 
 interface LinkField {
+  /**
+   * Defines referential integrity behavior when linked records are modified.
+   */
   actions?: {
+    /**
+     * Action to take when the referenced record is deleted.
+     */
     onDelete?: 'cascade' | 'restrict' | 'set null' | 'no action';
+
+    /**
+     * Action to take when the referenced record's key is updated.
```

Comment:
```suggestion
     * Action to take when the referenced record is updated.
```

Ensures consistency with the description of the property above.